test/triples.csv

```diff
@@ -90,4 +90,6 @@ geoId/44,typeOf,State,
 geoId/45,typeOf,State,
 geoId/46,typeOf,State,
 geoId/47,typeOf,State,
-geoId/48,typeOf,State,
```

Comment:
why removing geoId/48,typeOf,State?

Reply:
Sorry this was my mistake, I added this back